dmscripts/bulk_upload_documents.py

```diff
@@ -0,0 +1,39 @@
+import os
+import re
+import sys
+sys.path.insert(0, '.')
+from dmutils.documents import get_countersigned_agreement_document_path
+
+
+def upload_file(bucket, dry_run, file_path, framework_slug):
+    supplier_id = get_supplier_id_from_framework_file_path(file_path)
+    upload_path = get_countersigned_agreement_document_path(framework_slug, supplier_id)
+    if not dry_run:
+        with open(file_path) as file_contents:
+            bucket.save(upload_path, file_contents, acl='public-read', move_prefix=None)
```

Comment:
We shouldn't be setting `public-read` for agreement files, since they shouldn't have a public link, we're always generating signed URLs for them.